mephisto/abstractions/architects/local_architect.py

```diff
@@ -43,6 +43,9 @@ class LocalArchitectArgs(ArchitectArgs):
         default="localhost", metadata={"help": "Addressible location of the server"}
     )
     port: str = field(default="3000", metadata={"help": "Port to launch the server on"})
+    server_type: str = field(
+        default="npm", metadata={"Help": "Type of server to run, node or Flask"}
```

Comment:
perhaps the default should be `"node"` here, as `npm` solely refers to the package manager

Reply:
agree!